packages/proxmark3/PKGBUILD

```diff
@@ -0,0 +1,34 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=proxmark3
+pkgver=v4.13441.r968.gbac58d2a6
+pkgrel=1
+pkgdesc='A general purpose RFID tool for Proxmark3 hardware.'
+arch=('x86_64' 'aarch64')
+groups=('blackarch' 'blackarch-misc')
```

Comment:
Maybe `blackarch-hardware` and/or `blackarch-wireless`. See https://gist.github.com/noraj/5045be32490f1161161e16fc5820066c#ba to list categories. Try avoiding using misc by default but only when no category fits.

Reply:
also see https://github.com/BlackArch/blackarch/blob/f27c5c54aaf4dfc1b1d4bbbd41110952dc5d7773/packages/proxmark/PKGBUILD#L7, there is the radio group.